Mathlib/Analysis/InnerProductSpace/l2Space.lean

```diff
@@ -165,6 +165,7 @@ theorem inner_single_left (i : ι) (a : G i) (f : lp G 2) : ⟪lp.single 2 i a,
   · subst h; rfl
   · simp
 
+open Classical in
```

Comment:
When the decidability assumption is used for the left hand side of the equality in the theorem, can you instead add a decidability assumption?